git_machete/github.py

```diff
@@ -126,8 +126,12 @@ def get_token_from_file_in_home_directory() -> Optional[str]:
 
 
 def __extract_failure_info_from_422(response: Any) -> str:
+    additional_msg: str = ''
     if response['message'] != 'Validation Failed':
-        return str(response['message'])
+        if 'Reviews may only be requested from collaborators.' in response['message']:
+            additional_msg = "\nThere are some invalid reviewers in .git/info/reviewers file. " \
+                             "Please remove them or all file to create a pull request without reviewers."
```

Comment:
_all file_?...

Reply:
Oh, _all_ never applies to a single thing (same as _wszystkie_ in Polish), I'd rather say _or remove the entire file_